api/collections.js

```diff
@@ -0,0 +1,95 @@
+'use strict';
+
+const QuoteCollection = require('../models/quoteCollection.js');
+const router = require('express').Router();
+const HTTPStatus = require('http-status');
```

Comment:
Models are named with capital first letter, api end points are not, QuoteCollection (model) vs quoteCollections (api end point), as for the order of including I agree